src/main/java/io/confluent/connect/jdbc/sink/JdbcSinkConfig.java

```diff
@@ -51,7 +51,12 @@
   );
 
   public static final String CONNECTION_URL = "connection.url";
-  private static final String CONNECTION_URL_DOC = "JDBC connection URL.";
+  private static final String CONNECTION_URL_DOC =
+      "JDBC connection URL.\n"
+          + "For example, ``jdbc:oracle:thin:@localhost:1521:orclpdb1``, "
+          + "``jdbc:mysql://localhost/high_performance_java_persistence``, "
```

Comment:
Make these shorter so they don't wrap in the documentation if possible.
```suggestion
          + "``jdbc:mysql://localhost/db_name``, "
```